src/one_dragon_qt/view/setting/setting_env_interface.py

```
@@ -74,6 +74,13 @@ def _init_basic_group(self) -> SettingCardGroup:
         self.screenshot_method_opt.value_changed.connect(lambda: self.ctx.init_controller())
         basic_group.addSettingCard(self.screenshot_method_opt)
 
+        self.force_active_window_switch = SwitchSettingCard(
+            icon=FluentIcon.FULL_SCREEN, title='运行时恢复游戏窗口焦点',
+            content='仅前台模式运行期间生效；需要切出游戏时请先暂停，多次恢复失败会最小化其他窗口',
```

Ruff (0.15.21)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)